### PR TITLE
feat(smtp): make server configurable [DP-26]

### PR DESCRIPTION
**Pull request by georgettica**
also sending username

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -22,13 +22,21 @@
 
 def send_email(subject, body, recipients=None):
     """send an email to the selected recipients with a given body context."""
+
+    # TODO: hack for now, validation should happen when program starts
+    email_from = os.getenv("EMAIL_FROM")
+    if not email_from:
```
Comment (lmilbaum) [inline app.py:28]: I would prefer using a default value then raising an exception

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: app.py (hunk 1 above)
Comment (georgettica) [reply app.py:28]: What is a default value for this?

Comment (lmilbaum) [reply app.py:28]: Maybe; batyam@redhat.com 
We need to verify that batyam ML exists.

Comment (georgettica) [reply app.py:28]: and even if write this.. what about this project being open source and generic?
should the default be an email@redhat.com or any email in particular?

Comment (lmilbaum) [reply app.py:28]: good point :-)

**Review by lmilbaum — APPROVED** [review]

**Comment by georgettica** [comment]
waiting to have the time of day to configure the cronjob to use these variables before merging

**Pull request merged**